### ISSUE TITLE
Skeleton2D is completely broken in 3.5 (+small ru localization regression)

### ISSUE THREAD

**Issue by anunknowperson** [godotengine/godot#66448]
### Godot version

v3.5.stable.official [991bb6ac7]

### System information

Windows 10, GLES2, RTX 3060 Ti

### Issue description

I followed this tutorial:
https://docs.godotengine.org/en/stable/tutorials/animation/2d_skeletons.html?highlight=skeleton

Everything works fine, but up to this point:
"You will notice that all bones raise an annoying warning about a missing rest pose. This means that it's time to set one. Go to the skeleton node and create a rest pose. "

When I click on the "Bind Rest Pose" button, the size and position of all bones start constantly resetting to 0,0 and nan, nan. The error "core/math/transform_2d.cpp:49 - Condition "det == 0" is true."
 is spammed to the console 

I checked to do the same action on exactly the same project in version v3.4.stable.official [206ba70f4], and everything worked absolutely fine there.

---------------------------------------------------------------------------------------------------------------------------------------------

Also, in 3.5 there was a small but immediately noticeable bug in Russian localization, but I don't know where to report it.

For some reason, in the word "OpenGL", "Open" is translated, which looks unnatural and wrong.
![image](https://user-images.githubusercontent.com/54563399/192297326-0060b631-1220-4111-b70b-0a2494eabca9.png)

3.4:
![image](https://user-images.githubusercontent.com/54563399/192297690-2789239a-344a-4c3d-9074-c2873f91f41c.png)

### Steps to reproduce

Follow the tutorial on the link above and get to the "Bind Rest Pose" step.

Or download the test project and just hit Bind Rest Pose on Skeleton

### Minimal reproduction project

[AnimationTutorial.zip](https://github.com/godotengine/godot/files/9646896/AnimationTutorial.zip)

**Comment by akien-mga** [comment]
> Also, in 3.5 there was a small but immediately noticeable bug in Russian localization, but I don't know where to report it.

You can fix the translations on https://hosted.weblate.org/projects/godot-engine/godot/

**Comment by akien-mga** [comment]
CC @godotengine/animation

**Comment by Calinou** [comment]
@anunknowperson Can you reproduce this in any of the [3.5 betas and RCs](https://downloads.tuxfamily.org/godotengine/3.5/) to determine when the regression started?

**Comment by anunknowperson** [comment]
> @anunknowperson Can you reproduce this in any of the [3.5 betas and RCs](https://downloads.tuxfamily.org/godotengine/3.5/) to determine when the regression started?

Apparently, the problem was already in beta 1.

**Comment by lawnjelly** [comment]
Note that the tutorial you should use for 3.5 is:
https://docs.godotengine.org/en/3.5/tutorials/animation/2d_skeletons.html
(just in case there is some difference)

Ages since I did this, but I loaded the MRP project and clicked skeleton, and there is no "Bind Rest Pose" button, under Skeleton2D there is "Reset to Rest Pose" and "Overwrite Rest Pose".

I am getting a bunch of warnings without doing anything:
```
 core/math/transform_2d.cpp:49 - Condition "det == 0" is true.
```

**Comment by anunknowperson** [comment]
> Note that the tutorial you should use for 3.5 is: https://docs.godotengine.org/en/3.5/tutorials/animation/2d_skeletons.html (just in case there is some difference)
> 
> Ages since I did this, but I loaded the MRP project and clicked skeleton, and there is no "Bind Rest Pose" button, under Skeleton2D there is "Reset to Rest Pose" and "Overwrite Rest Pose". Maybe this is a translation issue if you are not using English, if you can let me know which button in English to look for?
> 
> I am getting a bunch of warnings without doing anything:
> 
> ```
>  core/math/transform_2d.cpp:49 - Condition "det == 0" is true.
> ```
> 
> When I click "overwrite rest pose" (which I'm guessing is what is intended, maybe the tutorial is out of date), the warnings go away, and there are no NaNs etc.
> 
> Interestingly, when I tried this in 3.4 stable (i.e. not a point release) this appears to be broken because clicking overwrite rest pose gives the determinant errors, and the bones disappear. Are you sure this is not broken in 3.4, and working in 3.5?

Yes, it looks like I really translated incorrectly. I followed the tutorial and pressed the top button, although since that time their order has changed.

Errors appear if you click on "Reset To Rest Pose" in the test project. In 3.4, "Reset To Rest Pose" works the same as "Overwrite Rest Pose" in 3.5. Therefore, I thought that the behavior of version 3.4 is correct.

That is, in 3.4 and 3.5, the behavior of buttons with the same text is reversed.
So errors appear if you press "Reset To Rest Pose" in 3.5 or "Overwrite Rest Pose" in 3.4.

Nevertheless, I think it's worth adding some kind of distinct error, for example "Rest Pose is not set".

**Comment by anunknowperson** [comment]
> I am getting a bunch of warnings without doing anything:
> 
> ```
>  core/math/transform_2d.cpp:49 - Condition "det == 0" is true.
> ```

The fact that about twenty errors are spammed immediately when the project is started, I did not even notice.  I was referring to the button click error spam I talked about in the post above. Then they spam non-stop and the size of all the bones is set to 0.0, which, when randomly moved, sets their position to nan, nan.
